rotkehlchen/tests/api/test_balances.py

```diff
@@ -761,13 +761,13 @@ def test_query_ksm_balances(rotkehlchen_api_server):
             result = assert_proper_response_with_result(response)
 
     # Check per account
-    account_1_balances = result['per_account']['KSM'][SUBSTRATE_ACC1_KSM_ADDR]
+    account_1_balances = result['per_account']['ksm'][SUBSTRATE_ACC1_KSM_ADDR]
     assert 'liabilities' in account_1_balances
     asset_ksm = account_1_balances['assets']['KSM']
     assert FVal(asset_ksm['amount']) >= ZERO
     assert FVal(asset_ksm['usd_value']) >= ZERO
 
-    account_2_balances = result['per_account']['KSM'][SUBSTRATE_ACC2_KSM_ADDR]
+    account_2_balances = result['per_account']['ksm'][SUBSTRATE_ACC2_KSM_ADDR]
     assert 'liabilities' in account_2_balances
     asset_ksm = account_2_balances['assets']['KSM']
```

Comment:
And I mean all places like that. Not only parts you touched in your code.

```suggestion
    account_2_balances = result['per_account']['ksm'][SUBSTRATE_ACC2_KSM_ADDR]
    assert 'liabilities' in account_2_balances
    asset_ksm = account_2_balances['assets'][A_KSM.identifier]
```